services/github/test/handler_test.js

```diff
@@ -747,6 +747,19 @@ helper.secrets.mockSuite(testing.suiteName(), [], function (mock, skipping) {
 
       assert(handlers.createTasks.calledWith({ scopes: sinon.match.array, tasks: sinon.match.array }));
     });
+
+    test('using collaborators_quiet policy should not create comment', async function () {
```

Comment:
I thought a comment should be created, but only once per PR?

Reply:
As I understand this policy and implementation is that it should not send comment if `_quiet` part of it is used.